docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
@gianm Glad to see JsonParser supporting falttening!

@himanshug I think what gianm said about "compatibility" is the grammer instead of implementation. Note that there is no ready-to-use JSON-path-like parser for avro, I think it's acceptable the grammer of flatternSpec differs in avro-version and JSON-version, for the ease of implementation.

Still don't you think the two-level-name-mapping when using schemaMapping with metrics is a little bit inconvenient...